templates/mediawiki-apb.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: servicecatalog.k8s.io/v1beta1
 kind: ServiceInstance
 metadata:
-  name: mediawiki123
```

Comment:
There's a limitation in CI where the deployment/deploymentconfig needs to have the name as the service instance resource so bind creds get delivered to the right place.

This needs to be mediawiki:
https://github.com/ansibleplaybookbundle/mediawiki-apb/blob/master/roles/provision-mediawiki123-apb/tasks/main.yml#L26